paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/PaymentSheetAnalyticsTest.kt

```diff
@@ -57,6 +57,7 @@ internal class PaymentSheetAnalyticsTest {
         validateAnalyticsRequest(eventName = "mc_load_started")
         validateAnalyticsRequest(eventName = "mc_load_succeeded")
         validateAnalyticsRequest(eventName = "mc_complete_sheet_newpm_show")
+        validateAnalyticsRequest(eventName = "elements.google_pay_repository.is_ready_request_api_call_failure")
```

Comment:
We could also turn off google pay for these tests.

Reply:
We always run an operation to check if the device is supported by Google Pay irregardless of if Google Pay is enabled in the config which is why this failure happens. I could disable this check in debug 😬

Reply:
Do we check even if google pay is disabled coming from elements/sessions?

Reply:
Yup! iOS does this as well with Apple Pay

Reply:
Could we return a different mocked response so that we don't do this then?